kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/FilterHandler.java

```diff
@@ -337,11 +360,11 @@ private void forwardRequest(DecodedRequestFrame<?> decodedFrame, RequestFilterRe
         // check it's a request
         String name = message.getClass().getName();
         if (!name.endsWith("RequestData")) {
-            throw new AssertionError("Attempt to use forwardRequest with a non-request: " + name);
+            throw new AssertionError("Filter '" + filterDescriptor() + "': Attempt to use forwardRequest with a non-request: " + name);
         }
 
         if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug("{}: Forwarding request: {}", channelDescriptor(), decodedFrame);
+            LOGGER.debug("{}: Filter '{}' forwarding request: {}", channelDescriptor(), filterDescriptor(), decodedFrame);
```

Comment:
I felt it was best to enclose the file names in single quotes to make clear that it's a name. Otherwise it can make the messages difficult to parse. For example in this case if the filter was called (for whatever reason) `error` then this log message would have been "Filter error forwarding request: ..." which is misleading.